v2/spiffetls/tlsconfig/config.go

```diff
@@ -73,14 +132,25 @@ func MTLSServerConfig(svid x509svid.Source, bundle x509bundle.Source, authorizer
 // wrapped by by this package and invoked after SPIFFE authentication has
 // completed.
 func HookMTLSServerConfig(config *tls.Config, svid x509svid.Source, bundle x509bundle.Source, authorizer Authorizer) {
-	panic("not implemented")
+	config.InsecureSkipVerify = true
```

Comment:
`InsecureSkipVerify` should be false for the server. The reason we have to use `true` on the client is that we want to bypass hostname validation of the server certificate. Hostname validation isn't performed against client certificates already.